toss.py

```diff
@@ -0,0 +1,182 @@
+    
+
+    totalTossWon=0
+    matchesWon=0
+    winPercentage=0
+
+    for matchId in listofMatchIds:
+        fileName=matchId.strip()
+        fileName=fileName+"_info.csv"
+        f=open("data/t20s_male_csv2/"+fileName,"r")
+        csv_f=csv.reader(f)
+        csv_f=list(csv_f)
+    
+        for info in csv_f:
+            if "toss_winner" in info:
+                team=""
+                for words in info[2:]:
+                    team+=words
+                if teamName == team:
+                    totalTossWon=totalTossWon+1
+                    break
+            if "winner" in info:
+                winner=""
+                for words in info[2:]:
+                    winner+=words
+                if teamName == winner:
+                    matchesWon=matchesWon+1
+                    break
+                else:
+                    break       
+        f.close()
+    
+    if(totalTossWon>0):
+        winPercentage=(matchesWon/totalTossWon)*100
+    else:
+        winPercentage=-1
+
+    return winPercentage
+
+def tossAverage():
+    teams=TeamAndVenues.keys()
+    totalTeams=len(teams)
+    tossWinPercent=0
+    for team in teams:
+        listofMatchIds=generateList(team,"")
+        win=tossWinPercentage(team,listofMatchIds)
+        if(win>=0):
+            tossWinPercent+=win
+        else:
+            totalTeams-=1
+    print("\nIf a team wins the toss, their match winning chances are(average %): ",tossWinPercent/totalTeams)
+
+def headToHead(team1,team2,listofMatchIds):
+    
+    team1MatchesWon=0
+    team2MatchesWon=0
+    for matchId in listofMatchIds:
+        fileName=matchId.strip()
+        fileName=fileName+"_info.csv"
+        f=open("data/t20s_male_csv2/"+fileName,"r")
+        csv_f=csv.reader(f)
+        csv_f=list(csv_f)
+    
+        for info in csv_f:
+            if "winner" in info:
+                team=""
+                for words in info[2:]:
+                    team+=words
+                if team1 == team:
+                    team1MatchesWon+=1
+                elif team2 == team:
+                    team2MatchesWon+=1
+        f.close()
+
+        team1win=team1MatchesWon/(team1MatchesWon+team2MatchesWon)*100
+        team2win=team2MatchesWon/(team1MatchesWon+team2MatchesWon)*100
+        
+        return [team1win,team2win]
+
+def teamRanking(team1, team2):
+    team1Rating=0
+    team2Rating=0
+    f=open("data/t20s_male_csv2/team_ranking.csv","r")
+    csv_f=csv.reader(f)
+    csv_f=list(csv_f)
+
+    for line in csv_f:
+        if team1 in line:
+            team1Rating=line[4:]
+        if team2 in line:
+            team2Rating=line[4:]
+        if team1Rating!=0 and team2Rating!=0:
+            break
+    f.close()
+    
+    team1Rating=int(team1Rating[0])
+    team2Rating=int(team2Rating[0])
+    team1win=team1Rating/(team1Rating+team2Rating)*100
+    team2win=team2Rating/(team1Rating+team2Rating)*100
+
+    return [team1win,team2win]
```

Comment:
A separate file for each feature would be a good practice.
Hence, lets extract this function into a new file `teamRanking.py`.